_posts/en/resources/2019-09-20-electronic-ultrasonic-sensor-arduino.md

```diff
@@ -0,0 +1,184 @@
+---
+layout:       article
+lang:         en
+parent:       electronic-resources
+breadcrumb:   true
+permalink:    /en/projects/electronic/ultrasonic-sensor-2019
+ref:          2020-ultrasonic-sensor-arduino
```

Comment:
2020 You're in advance on the calendar ;)

Reply:
Thanks! Sorry, I'm time travelling again